contentful_management/webhooks_call_proxy.py

```diff
@@ -0,0 +1,36 @@
+from .client_proxy import ClientProxy
+from .webhook_call import WebhookCall
+
+
+class WebhooksCallProxy(ClientProxy):
```

Comment:
WebhookResourceProxy is for proxies that are used from within the Webhook class, this one is the one used within the Client class.